### ISSUE TITLE
Unable to prune B2 on 2.0.10

### ISSUE THREAD

**Issue by trunet** [gilbertchen/duplicacy#325]
Works fine on 2.0.9. Don't know if related to #323 

```
Storage set to b2://duplicacy-proxmox1
Keep no snapshots older than 90 days
Keep 1 snapshot every 30 day(s) if older than 30 day(s)
Keep 1 snapshot every 7 day(s) if older than 7 day(s)
Deleting snapshot proxmox1 at revision 30
Deleting snapshot proxmox1 at revision 50
Deleting snapshot proxmox1 at revision 52
Deleting snapshot proxmox1 at revision 53
Failed to download the chunk 954c51e26f5fb6c99c8d25651beeefc84b54661a9ff4b11c96cc13a2c85c71e2: http: read on closed response body
```

**Comment by jt70471** [comment]
You may have to delete snapshot 53 and or possibly subsequent snapshots. Is this a backup destination or do you backup to another location then copy to b2?

**Comment by trunet** [comment]
this is the backup destination. It was fixed using a 2.1.0 provided by support.
